lib/prompts/rawlist.js

```diff
@@ -132,7 +131,6 @@ Prompt.prototype.onError = function() {
   this.hasError = true;
   this
     .error("Please enter a valid index")
-    .write( clc.bol(0, true) )
```

Comment:
@SBoudrias This actually fixes a bug where the message above was not shown on error.